railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -12,10 +12,10 @@
     <%- else -%>
       <%- if gemfile_entries.any? { |m| m.name == 'turbolinks' } -%>
     <%%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
```

Comment:
[As of today, there is no `async` attribute available on the `link` attribute.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link). I think this is because `link` elements do not block the document parser, they only block rendering.

I think it's probably sane that rendering blocks on CSS being downloaded. Otherwise you'd have some terrible FOUC issues.